### PR TITLE
[Meshery] Integrations: add Knative

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR adds Knative

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 73bd5513a493062b87ffc557133aa0279754c2d0 at: https://634daaab2d0196161b59a84e--layer5.netlify.app

**Comment by l5io** [comment]
You're amazing, @leecalcote

**Comment by sahilsuman933** [comment]
Please, @leecalcote, If we don't have the dark theme icon, leave the `darkModeIntegrationIcon` metadata. If you provide that, it will not apply the CSS filter property on the SVG.

**Comment by leecalcote** [comment]
@sahilsuman933 sure. Will do. Can you rephrase these instructions, though? I'm not entirely sure what you'd like the process to be.

**Comment by leecalcote** [comment]
Oh, I see: if no dark mode icon is available, simply leave the value empty for `darkModeIntegrationIcon`, in which case a CSS filter will be applied to stylize the color icon into a dark mode-friendly version. Ok, got it.
